modules/vpn/main.tf

```diff
@@ -91,7 +87,7 @@ module "vpn_server" {
   key_name                    = var.vpn_key_pair
   associate_public_ip_address = true
   vpc_security_group_ids      = [module.security_group_vpn.security_group_id]
-  user_data                   = join("", data.template_file.pritunl[*].rendered)
+  user_data                   = templatefile("${path.module}/scripts/pritunl-vpn.sh")
```

Comment:
I'm not really sure why template_file or templatefile is actually needed. There is nothing being replaced in this file from what I can see.